sequence/sequences.go

```diff
@@ -42,4 +42,26 @@ var Sequences = []Sequence{
 		},
 		LearningLinks: nil,
 	},
+	{
+		Name:        "Dirty Write",
+		Description: "Demonstrates a dirty write scenario where one transaction overwrites uncommitted changes of another transaction across two related tables",
```

Comment:
Demonstrates a lost update scenario where one transaction overwrites changes of another transaction across two related tables